engine/src/flutter/lib/web_ui/lib/src/engine/renderer.dart

```diff
@@ -59,6 +59,9 @@ abstract class Renderer {
   // Listens for view disposal events from the view manager.
   late StreamSubscription<int> _onViewDisposedListener;
 
+  /// Set the maximum number of bytes that can be held in the GPU resource cache.
+  set resourceCacheMaxBytes(int bytes);
```

Comment:
If skwasm also has `resourceCacheMaxBytes` then the implementation should be moved into this class since it will be the same for both renderers.